.idea/workspace.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ChangeListManager">
+    <list default="true" id="1780e761-075e-4461-956d-523b71e3e3d2" name="Changes" comment="">
+      <change beforePath="$PROJECT_DIR$/.pnp.cjs" beforeDir="false" afterPath="$PROJECT_DIR$/.pnp.cjs" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/packages/sources/alphavantage/package.json" beforeDir="false" afterPath="$PROJECT_DIR$/packages/sources/alphavantage/package.json" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/packages/sources/alphavantage/src/index.ts" beforeDir="false" afterPath="$PROJECT_DIR$/packages/sources/alphavantage/src/index.ts" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/yarn.lock" beforeDir="false" afterPath="$PROJECT_DIR$/yarn.lock" afterDir="false" />
+    </list>
+    <option name="SHOW_DIALOG" value="false" />
+    <option name="HIGHLIGHT_CONFLICTS" value="true" />
+    <option name="HIGHLIGHT_NON_ACTIVE_CHANGELIST" value="false" />
+    <option name="LAST_RESOLUTION" value="IGNORE" />
+  </component>
```

Comment:
What is this for?

Reply:
This is the folder that is generated by my idea (webstorm). I know you all probably using vscode. I guess we either need to add this to gitignore, or just include it as well (like we did for vscode). Currently i'm manually removing this folder each time, but it gets regenerated again.

Reply:
removed again and pushed, but i guess we need to solve this another way

Reply:
Got it. If we follow the precedent of `vscode`, I think we should include it then

Reply:
Or just update the `gitignore`. I'll leave it up to your discretion